docs/security-model.mdx

```diff
@@ -183,3 +136,41 @@ Notable mentions are:
 1. [Safari](https://www.infoq.com/news/2020/04/safari-third-party-cookies-block/)
 2. [Chrome](https://www.adexchanger.com/privacy/google-will-limit-cross-site-tracking-in-chrome-by-default-starting-in-february/)
 3. [Firefox](https://www.theverge.com/2019/6/4/18650363/firefox-block-trackers-default-enhanced-tracking-protection)
+
+## What about JSON Web Tokens?
+
+Currently, Ory Identities doesn't support JSON Web Tokens (JWTs) natively.
+
+:::note
+
+Do you need JWTs in your implementation? Let us know in
+[this issue for Ory Identities (Kratos)](https://github.com/ory/kratos/issues/2487).
+
+:::
+
+[Ory Zero Trust Identity and Access Proxy (Ory Oathkeeper)](https://www.ory.sh/oathkeeper) can "convert" Ory Sessions to JSON Web
```

Comment:
Is this naming for Oathkeeper correct? I've seen `Zero-Trust Proxy` in some docs, not sure about the "identity" part (but the hyphen in "zero-trust" doesn't look right).

Reply:
Yes that is correct https://www.google.com/search?client=safari&rls=en&q=iap+proxy&ie=UTF-8&oe=UTF-8